Assets/Scripts/DayNightCycle/TimeSwitcher.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Assets.Scripts.Services;
+using UnityEngine;
+using UnityEngine.UI;
+using Assets.Scripts.Settings;
+using UnityEngine.EventSystems;
+
+namespace Assets.Scripts.DayNightCycle
+{
+    public class TimeSwitcher : MonoBehaviour, IDragHandler
+    {
+
+        public Text Label;
+        public Slider TimeSlider;
+        public InputField TimeInput;
+        public Toggle ToggleDayNight;
+        DayAndNightControl dayNightCycle;
+
+       void Start()
+       {
+            
+            ToggleDayNight = GameObject.Find("ToggleDayNightCycle").GetComponent<Toggle>();
```

Comment:
Once again, this ToggleDayNightCycle appears to always be present. Why use .Find instead of a reference via the inspector?

Reply:
Will do